target/scripts/update-check.sh

```diff
@@ -0,0 +1,60 @@
+#! /bin/bash
+
+VERSION=$(</VERSION)
+#VERSION_URL="https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/master/VERSION" # doesn't exist yet
+VERSION_URL="https://raw.githubusercontent.com/casperklein/archive_docker-mailserver/update-check/VERSION"
+CHANGELOG="https://github.com/docker-mailserver/docker-mailserver/blob/master/CHANGELOG.md"
+
+_log() {
+  DATE=$(date '+%F %T')
+  echo "${DATE} ${1}"
+}
```

Comment:
We could in-line this function to be

```
_log { DATE=$(date '+%F %T'); echo "${DATE} ${1}"; }
```

which adheres to the styling guide in CONTRIBUTING.md ;)

Reply:
> which adheres to the styling guide in CONTRIBUTING.md

That functions have to be in one line is new for me? I did it for readability.

> maybe we want to add this to helper-functions.sh

It's a from case to case decision. In that case I thought, sourcing the complete helper file just for a little echo function would be overkill.
But I am open for discussion.

Reply:
> That functions have to be in one line is new for me? I did it for readability.

To be precise, you're right. One-line functions do not appear in CONTRIBUTING.md. But the way the function looks ATM, it does not adhere to the style guide as well :)

Reply:
Ack. I'll fix the style with my last commit.